scvi/models/modules.py

```diff
@@ -1,13 +1,52 @@
 import collections
-from typing import Iterable
+from typing import Iterable, List
 
 import torch
 from torch import nn as nn
 from torch.distributions import Normal
+from torch.nn import ModuleList
 
 from scvi.models.utils import one_hot
 
 
+def reparameterize_gaussian(mu, var):
+    return Normal(mu, var.sqrt()).rsample()
+
+
+class ConditionalInstanceNormalization(nn.Module):
```

Comment:
Did this end up helping? If not, should we still add it to the codebase?

Reply:
It was not helping much for GimVI, maybe it could be useful for further scVI researches. I don't have any preferences for the codebase.